CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ project(
         "The 'build-all' instructions for all examples for the book CMake Best Practices"
     LANGUAGES CXX
 )
-
+enable_testing()
```

Comment:
One of the enable_testing() calls is redundant in these changes.

Quoting from CMake documentation about enable_testing:
<https://cmake.org/cmake/help/latest/command/enable_testing.html>
```
Enable testing for current directory and below.
```

Reply:
Yes it is redundant, but in this case I added it specifically to the example to make it explicitly, so the example can be run standalone